app/views/external_forms/loader.js.erb

```diff
@@ -103,8 +96,9 @@
     var data = {};
     traverseNodeList(form.querySelectorAll("input"), function(input){
       if(input.name !== ""){
+        if (input.type === 'radio' && !input.checked) return;
         data[input.name] = input.value;
-        if (input.type == 'checkbox') {
+        if (input.type === 'checkbox') {
```

Comment:
Tell me you have experiences weird JS-behaviour without telling me... :-) :heart: